src/python/foglamp/core/api.py

```diff
@@ -38,20 +39,83 @@
 
     | GET             | /foglamp/statistics                                       |
     | GET             | /foglamp/statistics/history                               |
-    ------------------------------------------------------------------------------
+    -------------------------------------------------------------------------------
 """
 
 
+def _find_process_info(process_name):
+    """Find Process info"""
+
+    process = [proc for proc in psutil.process_iter() if proc.name() == process_name]
+
+    if len(process) == 0:
+        return None
+
+    return dict({
+        "pid": process[0].pid,
+        "status": process[0].status(),
+        "start_time": process[0].create_time(),
+        "cpu_percent": process[0].cpu_percent(interval=1.0),
+        "memory_percent": process[0].memory_percent(),
+        "system_disk_usage": psutil.disk_usage('/')
+    })
+
+
+def _prepare_process_info(_now, since_started):
+    """Prepare Json for detailed info"""
+
+    process_info = _find_process_info('foglamp')
```

Comment:
Why do we retrieve info for this process name? We have multiple services, either we provide a full set of info, this one alone would be irrelevant.